firebase-firestore/src/test/java/com/google/firebase/firestore/UserDataWriterTest.java

```diff
@@ -259,12 +257,10 @@ private static ObjectValue fromMap(Object... entries) {
   @Test
   public void testConvertsNestedObjects() {
     FieldValue actual = wrapObject("a", map("b", map("c", "foo"), "d", true));
-    ObjectValue expected =
-        fromMap(
-            "a",
-            fromMap(
-                "b", fromMap("c", StringValue.valueOf("foo")), "d", BooleanValue.valueOf(true)));
-    assertEquals(expected, actual);
+    ObjectValue.Builder expected = ObjectValue.newBuilder();
+    expected.set(field("a.b.c"), valueOf("foo"));
```

Comment:
Optional: You could chain these (`ObjectValue expected = newBuilder.set.set.build()`)

Reply:
It felt cleaner to spread these assignments over multiple lines. Kept as is.